command/secrets_enable.go

```diff
@@ -214,8 +214,7 @@ func (c *SecretsEnableCommand) Flags() *FlagSets {
 		Name:   flagNameAllowedManagedKeys,
 		Target: &c.flagAllowedManagedKeys,
 		Usage: "Managed key name(s) that the mount in question is allowed to access. " +
-			"Note that multiple keys may be specified either by providing the key names " +
-			"as a comma separated string or by providing this option multiple times, " +
+			"Note that multiple keys may be specified by providing this option multiple times, " +
```

Comment:
I've updated the docs here as specifying a comma separated list through the pi did not do the right thing (one string with the comma in it, not separate keys within the array).